WalletWasabi.Fluent/ViewModels/Wallets/Send/LabelSelectionViewModel.cs

```diff
@@ -73,46 +69,11 @@ public Pocket[] AutoSelectPockets(SmartLabel recipient)
 			return pockets;
 		}
 
-		if (knownPockets.Sum(x => x.Amount) >= _targetAmount)
-		{
-			return knownPockets;
-		}
-
-		if (unknownPockets.Sum(x => x.Amount) >= _targetAmount)
-		{
-			return unknownPockets;
-		}
-
-		if (NonPrivatePockets.Sum(x => x.Amount) >= _targetAmount)
-		{
-			return NonPrivatePockets;
-		}
-
-		if (privateAndKnownPockets.Sum(x => x.Amount) >= _targetAmount)
-		{
-			return privateAndKnownPockets;
-		}
```

Comment:
Should this be really removed? If it is enough combined with the known pockets why should we also add the semi private coins?

Reply:
Because using semi-private coins is better than using non-private coins (known).

Let's say I want to send 2 btc, and I have 1 btc private, 0,5 btc semi-private, and two 0,6 btc non-private.
So I can make a transaction as following:

1. 1 btc private + 0,5 btc semi-private + 0,6 btc non-private
2. 1 btc private + two 0,6 btc non-private

The first transaction is more private than the second one. https://github.com/zkSNACKs/WalletWasabi/issues/9092#issuecomment-1246498557

Reply:
I understand, thanks